workflows/sock-shop-demo/usingCmdProbe/workflow.yaml

```diff
@@ -716,6 +346,79 @@ spec:
         image: litmuschaos/litmus-checker:latest
         args: ["-file=/tmp/chaosengine.yaml","-saveName=/tmp/engine-name"] 
 
+    - name: disk-fill
+      inputs:
+        artifacts:
+          - name: disk-fill
+            path: /tmp/chaosengine.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                kind: ChaosEngine
+                metadata:
+                  name: catalogue-disk-fill
+                  namespace: {{workflow.parameters.adminModeNamespace}}
+                spec:
+                  appinfo:
+                    appns: 'sock-shop'
+                    applabel: 'name=catalogue-db'
+                    appkind: 'statefulset'
+                  annotationCheck: 'false'
+                  engineState: 'active'
+                  chaosServiceAccount: litmus-admin
+                  monitoring: false
+                  jobCleanUpPolicy: 'retain'
+                  components:
+                    runner:
+                      imagePullPolicy: Always
+                  experiments:
+                    - name: disk-fill
+                      spec:
+                        probe:
+                        - name: "check-catalogue-db-cr-status"
+                          type: "k8sProbe"
+                          k8sProbe/inputs:
+                            command:
+                              group: ""
+                              version: "v1"
+                              resource: "pods"
+                              namespace: "sock-shop"
+                              fieldSelector: ",status.phase=Running"
+                              labelSelector: "name=catalogue-db"
+                          operation: "present"
+                          mode: "Continuous"
+                          runProperties:
+                            probeTimeout: 1
+                            interval: 1
+                            retry: 1
```

Comment:
probePollingInterval?

Reply:
I refer docs
```
probe:
- name: "check-app-cluster-cr-status"
  type: "k8sProbe"
  k8sProbe/inputs:
    command:
      group: "<appGroup>"
      version: "<appVersion>"
      resource: "<appResource>"
      namespace: "default"
      fieldSelector: "metadata.name=<appResourceName>,status.phase=Running"
      labelSelector: "<app-labels>"
  operation: "present" # it can be present, absent, create, delete
  mode: "EOT"
  runProperties:
    probeTimeout: 5
    interval: 5
    retry: 1
   ```